coffee-rest/src/main/java/hu/icellmobilsoft/coffee/rest/validation/xml/JaxbTool.java

```diff
@@ -150,7 +200,7 @@
             throw new XsdProcessingException(CoffeeFaultType.WRONG_OR_MISSING_PARAMETERS, "type or inputStream is null!");
         }
         IXsdValidationErrorCollector errorCollector = createCDIInstance(IXsdValidationErrorCollector.class);
-        try {
+        try (inputStream) {
```

Comment:
nem vagyok biztos benne hogy egy parameterben kapott inputStream-et piszkalni kellene barhogy is